contracts/AssetToken.sol

```diff
@@ -86,6 +96,18 @@ contract AssetToken is ERC223Interface, IERC20 {
         return _balances[who];
     }
 
+
+    function setEmergencyPermission(address who) public onlyOwner {
+        if(_emergencyDelegated != who){
```

Comment:
I would just set _emergencyDelegated to whatever is passed in, just to keep the code simpler